src/search/components/SearchResults.js

```diff
@@ -7,16 +7,42 @@ export default class SearchResults extends React.Component {
   }
 
   static defaultProps = {
-    query: '',
+    query: null,
     results: [],
   }
 
   render() {
     const { query, results } = this.props;
+    const searchResultsClass = 'search-results';
+
+    let searchResults;
+
+    if (query && results.length > 0) {
+      searchResults = (
+        <div className={searchResultsClass}>
+          <p ref="message">Your search for "{query}" returned {results.length} results.</p>
+          <ul ref="results">
+            {results.map((result) => <li key={result.title}>{result.title}</li>)}
+          </ul>
+        </div>
+      );
+    } else if (query && results.length === 0) {
+      searchResults = (
+        <div className={searchResultsClass}>
+          <p ref="message">No results were found for "{query}".</p>
+        </div>
+      );
+    } else if (query !== null) {
+      searchResults = (
+        <div className={searchResultsClass}>
+          <p ref="message">Please supply a valid search</p>
+        </div>
+      );
+    }
+
     return (
-      <div className="search-results">
-        <h3>Your search for "{query}" returned {results.length} results.</h3>
-        {results.map((result) => <div key={result.title}>{result.title}</div>)}
+      <div ref="container" className="search-results">
+        {searchResults}
```

Comment:
It looks like this would currently render as:

```html
<div class="search-results" /* ref="container" */>
  <div class="search-results">
    <p /* ref="message" */>Your search...</p>
    ...
  </div>
</div>